tox.ini

```diff
@@ -40,3 +40,8 @@ envlist =
     black-{python2.7,python3.7}
     {py27,py37}-{isort,flake8,pytest}
 skipdist = true
+
+[gh-actions]
+python =
+    2.7: py27-{isort,flake8,pytest}
+    3.7: black-python2.7,black-python3.7,py37-{isort,flake8,pytest}
```

Comment:
black-python2.7 needs to run on python3.7?

Reply:
Yes, you can't install black on a py27 virtualenv. The package is python3-only. It prints

```
ERROR: Could not find a version that satisfies the requirement black (from versions: none)
ERROR: No matching distribution found for black
```

Reply:
Black isn't a package, though, it's a command. It can analyze 2.7 code, but the program runs in 3.x. You don't care if "ls" runs in python or C, same thing here. "black" runs in python 3.x, but it analyzes any version of code.